samples/excel/85-preview-apis/data-types-formatted-number.yaml

```diff
@@ -0,0 +1,149 @@
+order: 1
+id: excel-data-types-formatted-number
+name: 'Data types: Formatted numbers'
+description: This sample shows how to set and get data types using the formatted number properties.
+host: EXCEL
+api_set:
+    ExcelApi: BETA (PREVIEW ONLY)
+script:
+    content: |
+        $("#set-formatted-number-date").click(() => tryCatch(setFormattedNumberDate)); 
+        $("#set-formatted-number-currency").click(() => tryCatch(setFormattedNumberCurrency)); 
+        $("#get-formatted-number").click(() => tryCatch(getFormattedNumber));
+
+        async function setFormattedNumberDate() {
+          await Excel.run(async (context) => {
+            // This method creates a formatted number data type,
+            // and sets the format of this data type as a date.
+
+            // Create a worksheet called "Sample" and activate it.
+            context.workbook.worksheets.getItemOrNullObject("Sample").delete();
+            const sheet = context.workbook.worksheets.add("Sample");
+            sheet.activate();
```

Comment:
When I was playing around with the sample, I missed this worksheet reset was here. Could this be moved to a "Setup" method and button? Alternatively, you could have the buttons target the currently selected cell, if you like the "setup-less" sample style.